frontend/src/pages/ApprovedActivityReport/index.js

```diff
@@ -386,6 +388,7 @@ export default function ApprovedActivityReport({ match, user }) {
               recipientType,
               'Reason',
               'Program Type',
```

Comment:
I know this is not your change, but should "Type" have a capital 'T'?

Reply:
Yep. Lauren & Kelly are preparing a list of capitalization for us to fix now that we've decided on a style, but totally make sense to fix this while I'm here.